apps/pkeyutl.c

```diff
@@ -20,12 +20,15 @@
 #define KEY_PUBKEY      2
 #define KEY_CERT        3
 
+static EVP_PKEY *get_pkey(const char *kdfalg,
+                          const char *keyfile, int keyform, int key_type,
+                          char *passinarg, int pkey_op, ENGINE *e);
 static EVP_PKEY_CTX *init_ctx(const char *kdfalg, int *pkeysize,
-                              const char *keyfile, int keyform, int key_type,
-                              char *passinarg, int pkey_op, ENGINE *e,
-                              const int impl, int rawin, EVP_PKEY **ppkey,
-                              EVP_MD_CTX *mctx, const char *digestname, const char *kemop,
-                              OSSL_LIB_CTX *libctx, const char *propq);
+                              int pkey_op, ENGINE *e,
+                              const int impl, int rawin,
+                              EVP_PKEY *pkey /* may get deallocated */,
```

Comment:
I don't like this change...

Reply:
Oops, I don't like it either ;-)
I forgot to copy over to here the comment I had already changed  in the function definition header; fixed now.